pulp_rpm/app/migrations/0026_checksum_null.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.16 on 2020-10-09 14:58
+
+from django.db import migrations, models
+from django.db.migrations import exceptions
+
+
+def check_checksum(apps, schema_editor):
+    checksum_model = apps.get_model("rpm", "Checksum")
+    q_bad_dist_tree = checksum_model.objects.filter(checksum=None)
+    if q_bad_dist_tree:
+        dist_tree_pks = {
+            c.distribution_tree.pk for c in checksum_model.objects.filter(checksum=None)
+        }
+        raise exceptions.AmbiguityError(
+            f"Please remove {dist_tree_pks} and re-sync after this migration."
```

Comment:
"Your Pulp instance contains DistributionTree content unis with null checksums. They were not properly validated before this update, please remove the following DistributionTree content and continue the upgrade. If you need to get removed content back, please re-sync corresponding repositories."
How does that sound? Any other suggestions?

Reply:
Way better than mine